ros2doctor/test/test_hello.py

```diff
@@ -14,10 +14,39 @@
 
 from argparse import Namespace
 
+from launch import LaunchDescription
+from launch.actions import ExecuteProcess
+from launch.actions import OpaqueFunction
+
+import launch_testing.markers
+
+import pytest
+
 from ros2doctor.verb.hello import HelloVerb
 from ros2doctor.verb.hello import SummaryTable
 
 
+@pytest.mark.rostest
+@launch_testing.markers.keep_alive
+def generate_test_description(ready_fn):
+    return LaunchDescription([
+        # Always restart daemon to isolate tests.
+        ExecuteProcess(
+            cmd=['ros2', 'daemon', 'stop'],
+            name='daemon-stop',
+            on_exit=[
+                ExecuteProcess(
+                    cmd=['ros2', 'daemon', 'start'],
+                    name='daemon-start',
+                    on_exit=[
+                        OpaqueFunction(function=lambda context: ready_fn())
```

Comment:
FYI, this is the old ([soon-to-be deprecated](https://github.com/ros2/launch/pull/346)) way of doing this. Use `launch_testing.actions.ReadyToTest()` instead.